handbook/ldap/ldap.md

```diff
@@ -0,0 +1,118 @@
+
+- Administrator desktop access to the SystemLink server.
+
+- An LDAP server accessible to the SystemLink server.
+
+- A bind user and bind password for the LDAP server.
+
+- Familiarity with the LDAP attributes available to your organization. If you do not know what LDAP attributes and groups are available to you please contact the system administrator for your LDAP server.
+
+## Enabling LDAP in SystemLink
+
+### Configuring Authentication
+
+1. Log into the server running SystemLink and open **NI Web Sever Configuration**.
+
+2. Go to the **Authentication** tab and enable **Connect to an LDAP server**.
+
+3. Enter the LDAP url for your sever. Refer to [**LDAP URLs**](#ldap-urls) for details on how to structure this URL.
+
+4. Enter the bind user. Refer to [**Bind User**](#bind-user) for details on usernames that can be used here.
+
+5. Enter the bind password.
+
+6. Click **Apply and Restart**.
+
+<figure>
+  <img src="../../img/ldap-setup.png" width="500" />
+  <figcaption>Configuration needed to enable logging into a SystemLink server using LDAP credentials.</figcaption>
+</figure>
+
+You may now login to the SystemLink server using your LDAP credentials. Please note that you will not be able to access systems and data in SystemLink unless you have configured workspace and role mappings. Refer to [**Mapping LDAP Attributes and Groups to SystemLink Workspaces and Roles**](#mapping-ldap-attributes-and-groups-to-systemlink-workspaces-and-roles) for details.
+
+<figure>
+  <img src="../../img/no-mapping.png" width="500" />
+  <figcaption>View from a user who can login but does not have any workspace and role mappings.</figcaption>
+</figure>
+
+## LDAP URLs
+
+LDAP URLs follow a standard scheme. The URL determines the server to connect to, the server's port, a base search DN, and an attribute that determines what users will provide as a username when logging into SystemLink.
+
+!!! note "Example LDAP URL and login window"
+
+    **Structure:** `ldap://<server-dns>:<port>/<target-entry-dn>?<username-attribute>`
+
+    **Example:** `ldap://example.com:389/dc=example,dc=com?sAMAccountName`
+
+    In this example the server is `example.com`, the port is 389, the base search DN is `dc=example,dc=com` and the LDAP attribute used for user login is `sAMAccountName`. 
+
+    <figure>
+      <img src="../../img/ldap-login.png" width="500" />
+      <figcaption>Username `jdoe` is used because the `sAMAccountName` attribute is specified in the LDAP URL.</figcaption>
+    </figure>
+
+If a username attribute is not specified **NI Web Server Configuration** will automatically add `uid` as the attribute. Depending on how your LDAP directory is setup this attribute may or may not be available.
+
+!!! note "Common username attributes"
+    - `uid`
+    - `userPrincipalName`
+    - `sAMAccountName`
+
+## Bind User
```

Comment:
The `LDAP URLs` and `Bind User` sections are so closely related it might be worth combining them. If you take my other suggestion of referring back to this section from `LDAP User Mapping`, I think the user will need to read through both anyway.

Reply:
I combined the sections and made subsections. I also considered the material some.